proxy/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM nginx:stable-alpine3.17-slim
+
+COPY nginx.conf /etc/nginx/nginx.conf
+COPY www /data/www
```

Comment:
Nitpick: I think the directory structure would make more sense with this folder being in the root of the repository (perhaps named `frontend` or `public` though `www` also works) rather than in the proxy's subdirectory.

Reply:
Moved to /static, note this will slightly increase build time since context is larger

Reply:
That reminds me... We should probably define appropriate `.dockerignore` as well. Contents of `.gitignore` + addition of `.git` directory should be enough, I guess? We can add `.gitignore`, `LICENSE`, and `README` files as well though they're unlikely to change often anyway.

Reply:
I see you force-pushed a change using the `additional_contexts` feature, that does make the suggestion about additional files (over what's in gitignore) irrelevant since they're not going to be part of the context anyway.

Reply:
Yup, those files are no longer part of the build context